src/patternfly/upgrade-examples/PopoverUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
```

Comment:
add in a comparison between `popovers-pf` and `pf-c-popover`.
and then `container-popovers-pf` with `  `. 
Or maybe do some further investigation into which classes are comparable, but make sure you include them all even if they don't directly correlate.

Reply:
- `popovers-pf` and `pf-c-popover` corrected
- `popover` switched with `container-popovers-pf`
- added popover directional arrow classes to the list of comparable classes.